firefox-ios/Client/Coordinators/Browser/BrowserNavigationHandler.swift

```diff
@@ -96,6 +97,9 @@ protocol BrowserNavigationHandler: AnyObject, QRCodeNavigationHandler {
 
     /// Shows the app menu
     func showMainMenu()
+
+    /// Navigates from home page to a new link
+    func navigateFromHomePanel(to url: URL, visitType: VisitType, isGoogleTopSite: Bool)
```

Comment:
this is specific to homepage today because I am calling the `homePanel` method in the browser coordinator. It would be great to have a single `navigateToLink` method instead, but leaving this as of now until we revisit some logic that we are doing in BVC. Let me know if any issues!